tpp-user-guide.md

```diff
@@ -650,6 +650,21 @@ curl --cert transport.pem --key transport.key -H 'Authorization: Bearer 2Pjwts8m
 
 ### 4.3.1 Create OpenID Connect Request Object
 
+Different methods of authentication (_private_key_jwt_ and _tls_client_auth_), of sending Request Object (with and without *PAR*) and response modes (with and without _JARM_) can be supported by _Authorization Servers_ according to the [FAPI_2_0_Advanced_Profile] specification. Therefore, how does it reinforce the security profile for Open Banking Brasil (item 8 of section 5.2.3 of the [security requirements for the confidential client](https://openbanking-brasil.github.io/specs-seguranca/open-banking-brasil-financial-api-1_ID3.html#section-5.2.3)), *all* methods shall be supported by the TTPs (API clients).
```

Comment:
Avaliar se não seria [FAPI-1-Advanced] e inserir o link para a URL

Reply:
Ok. What does "BR-OB Adv. OP w/ MTLS, PAR, JARM" OIDF certification profile means? Do "Banco Agile" and "Banco Arbi" (for example) just support JARM? As we don´t have this requirement (at least one without JARM), how do we going to guarantee that all TTP´s could "talk" to all certificated Banks?

Reply:
no they don't 'just support JARM', their OpenID provider supports JARM and not JARM. They just haven't understood that they MUST certify a non JARM configuration and they can optionally (MAY) choose to certify as well a JARM configuration. 

The specification is clear. This is an issue really for squad sandbox as they need to contact both banks and tell them to certify one of the 4 mandatory profiles. They can keep advertising support for JARM as well if they want.

Reply:
Great, so I will take this point to Sec WG review this directive.

Reply:
To avoid any confusion - add this to the authorisation server requirements.

* shall support the response_type value code id_token;
* may support response_type value code in conjunction with the response_mode value jwt;